lib/msgraph/models.go

```diff
@@ -74,6 +74,24 @@ type Application struct {
 	OptionalClaims        *OptionalClaims `json:"optionalClaims,omitempty"`
 }
 
+const (
+	// OPTIONAL_CLAIM_GROUP_NAME is the group claim.
+	OPTIONAL_CLAIM_GROUP_NAME = "groups"
+)
+
+const (
+	// OPTIONAL_CLAIM_ADDITIONAL_PROPERTIES_SAM_ACCOUNT_NAME is the sAMAccountName claim.
```

Comment:
Is "sAMAccountName" correct? Should it be SAMAccountName?

Same for other lines.

Reply:
yes, https://learn.microsoft.com/en-us/entra/identity/hybrid/connect/how-to-connect-fed-group-claims#group-claims-for-applications-migrating-from-ad-fs-and-other-identity-providers

Reply:
Weird 🤷‍♂️. Thanks for the link.